packages/validator/templates/configSecurity.json

```diff
@@ -0,0 +1,98 @@
+{
+  "config": {
+    "ois": {
+      "__arrayItem": {
+        "__oisSpecs": {}
+      }
+    },
+    "triggers": {
+      "__optional": {
+        "request": {
+          "__arrayItem": {
+            "endpointId": {
+              "__regexp": "^0x[a-fA-F0-9]{64}$"
+            },
+            "oisTitle": {},
+            "endpointName": {}
+          }
+        }
+      }
+    },
+    "nodeSettings": {
+      "providerIdShort": {
+        "__regexp": "^[a-fA-F0-9]{7}$"
+      },
+      "nodeVersion": {
+        "__regexp": "^([0-9]*\\.){2}([0-9]*)$"
```

Comment:
nit: this regex matches strings like `..1`.

@bbenligiray are we doing something like this: https://www.geeksforgeeks.org/introduction-semantic-versioning/

If so, I think it would be as simple as this: "^[0-9]+\\.[0-9]+\\.[0-9]+$"

assuming we don't have trailing identifiers, e.g. `1.0.0-alpha.1`

Reply:
Actually we probably want to support trailing identifiers, especially for node versions customized for some other chain for example.

Reply:
You could create a regex from the grammar from the original specs: https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions

Although it looks like a bit of pain in the ass.

Reply:
There's a regex in the official specs for semantic versioning: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

Reply:
Looks good (I guess we would use the second one)

Reply:
I assume this should be replaced across all the templates -> added to [template issue](https://github.com/api3dao/airnode/issues/197)